src/pyload/plugins/downloaders/SKnigavuhe.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from ..base.simple_downloader import SimpleDownloader
+
+
+class SKnigavuhe(SimpleDownloader):
+    __name__ = "SKnigavuhe"
```

Comment:
class name and the python filename has to be called `KnigavuheOrg` and `KnigavuheOrg.py`  accordingly
`__name__` has to be called `KnigavuheOrg`